core/src/test/scala/org/apache/spark/sql/delta/DeltaErrorsSuite.scala

```diff
@@ -423,9 +422,15 @@ trait DeltaErrorsSuiteBase
            |Your table's current Delta protocol version:
            |Protocol(2,6)
            |
-           |Please upgrade your table's protocol version using $cmd and try again.
+           |Please upgrade your Delta table to reader version 2 and writer version 5
+           |and change the column mapping mode to 'name' mapping. You can use the following command:
            |
-           |""".stripMargin)
+           |ALTER TABLE <table_name> SET TBLPROPERTIES (
+           |   'delta.columnMapping.mode' = 'name',
+           |   'delta.minReaderVersion' = '2',
+           |   'delta.minWriterVersion' = '5')
+           |
+    """.stripMargin)
```

Comment:
nit: either indent this another 2 spaces (line up with the string start) or move to the above line